core/indexer/elasticsearchDatabaseClient.go

```diff
@@ -166,7 +168,7 @@ func closeESResponseBody(res *esapi.Response) {
 	if res != nil && res.Body != nil {
 		err := res.Body.Close()
 		if err != nil {
-			log.Trace("error closing elastic search response body", "error", err)
+			log.Warn("error closing elastic search response body", "error", err)
```

Comment:
"cannot close elastic search response body" is enough, error is displayed in err

Reply:
done.